src/modules/aws-ec2/iam-instance-profile/getIamInstanceProfileAssociations.ts

```diff
@@ -0,0 +1,16 @@
+import { EC2 } from "aws-sdk";
+export const getIamInstanceProfileAssociations = async (instanceId: string) => {
+  const ec2 = new EC2();
+  const params = {
+    Filters: [
+      {
+        Name: "instance-id",
+        Values: [instanceId]
+      }
+    ]
+  };
+  const responseData = await ec2
+    .describeIamInstanceProfileAssociations(params)
+    .promise();
+  return responseData;
```

Comment:
If we don't use the return value from `describeIamInstanceProfileAssociations`, we can just return the promise and omit the async/wait keywords from the function.